sys/linux/dev_msm.txt

```diff
@@ -0,0 +1,147 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
```

Comment:
Please add a link to source code of the driver, so that's it's possible to find it later and understand what version was used.
I see upstream has these consts, but it does not seem to have any implementation:
https://elixir.bootlin.com/linux/v5.12-rc8/A/ident/DRM_IOCTL_MSM_SUBMITQUEUE_QUERY